src/processTargets/modifiers/ItemStage/tokenizeRange.ts

```diff
@@ -0,0 +1,162 @@
+import { Range, TextEditor } from "vscode";
+
+/**
+ * Takes the range for a collection and returns a list of tokens within that collection
```

Comment:
Please define  what a "token" is here, as we use the term "token" all over the place and this is not what we mean.  Tbh I'm tempted to not use "token" at all as a term here given that it violates our naming conventions, but at the very least it would be good to have a comment indicating how you're using the term here

An example would prob be good; see what I did on lines 19-20 below